addons/event/static/src/client_action/event_barcode.scss

```diff
@@ -24,6 +24,30 @@
         max-width: 200px;
         max-height: 100px;
     }
+    .o_barcode_mobile_container {
+        margin-top: 75px;
+        margin-bottom: -35px;
+        img {
+            height: 185px;
+            width: 275px;
+        }
+        // In order to have the o_mobile_barcode button on both the image and the label,
+        // We use negative margin at the bottom and 0 opacity (since not needed in the view)
```

Comment:
Note for review: this is a trick. It overlaps the label, making the whole area clickable.